pareto/utilities/get_data.py

```diff
@@ -73,7 +73,7 @@ def _read_data(_fname, _set_list, _parameter_list):
 
     _set_list = list(set(_set_list))
     _data_column = list(set(_data_column))
-    generic_words = ["index", "nodes", "time"]
+    generic_words = ["index", "nodes", "time", "pads"]
```

Comment:
I believe "storage" should be added here? In case the tabe "StorageInitialWaterQuality" has data

Reply:
I don't think this is needed. I'm thinking the "StorageInitialWaterQuality" tab will be structured similarly to "InitialStorageCapacity" tab in the LAYFLAT file and  have an index column called "StorageSites" as the index. I believe it will be read in okay if the index column is "StorageSites". The "StorageInitialWaterQuality" tab is left blank for now because we don't currently have any StorageSites in the generic operational model, so it reads incorrectly if I fill in the column titles.

Reply:
As discussed, to look into how to make this work both without / with entries on sheet "StorageInitialWaterQuality".